kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/htm-applications.yaml

```diff
@@ -0,0 +1,26 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: htm-applications
```

Comment:
@suppathak this seems unrelated to this PR, can you remove the changes to jupyterhub?